internal/scorecard/alpha/config.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"log"
+
+	"gopkg.in/yaml.v2"
+)
+
+type ScorecardTest struct {
+	Image       string            `yaml:"image"`                // The container image name
+	Entrypoint  string            `yaml:"entrypoint,omitempty"` // An optional entrypoint passed to the test image
+	Labels      map[string]string `yaml:"labels"`               // User defined labels which are used by scorecard to filter tests
+	Description string            `yaml:"description"`          // User readable test description
+}
+
+// ScorecardConfig represents the set of test configurations which scorecard
+// would run based on user input
+type ScorecardConfig struct {
+	Tests []ScorecardTest `yaml:"tests"`
+}
+
+// String returns the yaml text representation of the ScorecardConfig
+func (c ScorecardConfig) String() string {
+	s, err := yaml.Marshal(&c)
+	if err != nil {
+		log.Fatalf(err.Error())
+	}
+	return string(s)
+}
```

Comment:
What's the plan for this? Ideally we wouldn't use logrus from a library-ish function/package, especially `log.Fatal`

Reply:
removed in latest commit, I carried that over from my testing of this, doesn't need to be there at this point.